src/circuits/etable_compact/op_configure/op_load.rs

```diff
@@ -87,10 +87,10 @@ impl<F: FieldExt> EventTableOpcodeConfigBuilder<F> for LoadConfigBuilder {
         let is_four_bytes = common.alloc_bit_value();
         let is_eight_bytes = common.alloc_bit_value();
         let is_sign = common.alloc_bit_value();
+        let is_i64 = common.alloc_bit_value();
```

Comment:
I guess we need a constraint between `is_i64` and `vtype`.

Reply:
right, my bad